config/angular/repos/NG-ZORRO-ng-zorro-antd.json

```diff
@@ -1,8 +1,8 @@
 {
   "version": 1,
   "source": "github",
-  "authorIds": [],
-  "owner": "NG-ZORRO",
+  "authorIds": ["yadongxie"],
+  "owner": "Yadong Xie",
```

Comment:
This is not a valid owner, owner must be the GitHub ID of the owner user / organization which can't be a name with spaces.